week2/2-website/index.html

```diff
@@ -1,21 +1,98 @@
-    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
-    <link rel="stylesheet" href="css/normalize.css">
-    <!-- Add a link to your CSS file here (use the line above to guide you) -->
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <link
+      href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
 
-</body>
+    <link rel="stylesheet" href="css/style.css" />
+    <!-- Add a link to your CSS file here (use the line above to guide you) -->
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo">
+          <img src="/img/karma-logo.svg" alt="Logo" width="26" />
+        </div>
+        <ul class="navlinks">
+          <li><a class="active" href="#">Meet Karma</a></li>
+          <li><a href="#">How it Works</a></li>
+          <li><a href="#">Store</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Help</a></li>
+          <li><a href="#">Login</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="intro">
+        <div class="container">
+          <h1>Introducing Karma</h1>
+          <h3>Bring WiFi with you, everywhere you go.</h3>
+          <div class="button">
+            <button>Learn More</button>
+          </div>
+        </div>
+      </section>
+      <section class="works">
+        <div class="container">
+          <h2>Everyone needs a little Karma.</h2>
+          <ul class="how">
+            <li>
+              <img
+                src="/img/icon-devices.svg"
+                alt="devices"
+                width="127"
+                height="127"
+              />
+              <h3>Internet for all Devices</h3>
+            </li>
+            <li>
+              <img
+                src="/img/icon-coffee.svg"
+                alt="productivity"
+                width="127"
+                height="127"
+              />
+              <h3>Boost your productivity</h3>
+            </li>
+            <li>
+              <img
+                src="/img/icon-refill.svg"
+                alt="refill"
+                width="127"
+                height="127"
+              />
+              <h3>Pay as You Go</h3>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="fcontainer">
```

Comment:
There's a border missing here. You'd want to add a `<hr>` element.

Reply:
Have now added the border and styling for it.